templates/base.html

```diff
@@ -102,6 +107,33 @@
         </script>
         {% endblock script %}
         {% include '_includes/stat_counter.html' %}
+
+        {% if (( PHOTOS_LIGHTBOX == "True" ) and (article and article.photo_gallery)) %}
+        <!-- 'photo' Gallery support -->
+        <!-- Magnific Popup core JS file -->
+        <script src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/magnific/magnific-popup.js"></script>
+        <script>
+            $('.gallery').magnificPopup({
```

Comment:
Just a question. Can we use lightbox without jquery?

Reply:
Quoting: Magnific Popup is a responsive lightbox & dialog script with focus on performance and providing best experience for user with any device
(for jQuery or Zepto.js).

Reply:
So I guess 'no' unless we use zepto.js